paasta_tools/deployd/watchers.py

```diff
@@ -214,7 +214,11 @@ def process_default(self, event):
             if new_config != self.public_config:
                 self.log.info("Public config has changed, now checking if it affects any services config shas")
                 self.public_config = new_config
-                service_instances = self.get_service_instances_with_changed_id()
+                instances = get_services_for_cluster(cluster=self.public_config.get_cluster(),
```

Comment:
given that this returns a list of pairs, wasn't this better named ``service_instances``?